project/ProjectUtils.scala

```diff
@@ -67,11 +58,25 @@ object ProjectUtils {
       Opt("--length", spec.length),
       Opt("--init", spec.init),
       Opt("--next", spec.next),
-      Opt("--cInit", spec.cInit),
+      //Opt("--cInit", spec.cInit), // cInit cannot be an empty string
```

Comment:
Not sure what's going on here. Should we  check for this and only include it if it is non-empty? Leaving it commented out seems like it might lead to confusion later.

Reply:
Ideally there would be a check, but I don't know if we could do this inside a `Cmd` parameter. The easy way is to have a choice between two Cmds, but I would like to avoid the duplication. Thoughts?

I left the `Opt` there exactly to avoid future confusion, i.e., to explain why it is not part of the default. Maybe it is better to have the text comment without the `Opt`.

Reply:
@shonfeder?

Reply:
Oops looo like more reply was left pending yesterday. Sorry!

Reply:
If you like my suggestion, you could do it with an empty string instead of an optional empty string of course.

Reply:
Cool! I didn't know about this interaction between Option and map. Will make the change.